packages/sitecore-jss/src/native-fetcher.ts

```diff
@@ -0,0 +1,81 @@
+import { HttpResponse } from './data-fetcher';
+import debuggers, { Debugger } from './debug';
+
+type NativeDataFetcherOptions = {
```

Comment:
We should allow to pass more arguments, b/c we expect that on the [line](https://github.com/Sitecore/jss/pull/1008/files#diff-04aaa9815a34b8a043be4f5d5cf48bd5060f6c2eaf2a2462be70d597c59c46d8R27) using init
```suggestion
type NativeDataFetcherOptions = {
  [key: string]: unknown
```

Reply:
We have this covered with `NativeDataFetcherConfig` (combination of `NativeDataFetcherOptions` and `RequestInit`):
https://github.com/Sitecore/jss/pull/1008/files#diff-04aaa9815a34b8a043be4f5d5cf48bd5060f6c2eaf2a2462be70d597c59c46d8R15